PiView_AG/Host.py

```diff
@@ -59,17 +61,19 @@ def name(self):
         """
         return gethostname()
 
-    def python(self):
+    @staticmethod
+    def python():
         """
         Get current Python version
 
         :rtype: string
         :return: A string containing the version of python that is being used
         """
-        pythonv = platform.python_version()
```

Comment:
`sys.platform` is a string, so it doesn't make sense to call `python_version()` on that. I have changed it to use `version_info` from `sys` in stead. Only using the first three fields, so it would return something like '3.9.1'.

Reply:
sounds good